javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -7,6 +7,44 @@ import { useAuth0 } from "@auth0/auth0-react";
 import { fetchWithToken } from "main/utils/fetch";
 import NavbarHover from "../Nav/NavbarHover";
 
+export function PureNavbar({ isAdmin, isMember, ChannelPages, AdminPages, SearchPages, DataPages }) {
```

Comment:
By convention, prop names are usually lowerCamelCase. Can you rename this?

Reply:
Done!